packages/aws-cdk/lib/init-templates/v2/app/typescript/cdk.template.json

```
@@ -1,3 +1,7 @@
 {
-  "app": "npx ts-node --prefer-ts-exts bin/%name%.ts"
+  "app": "npx ts-node --prefer-ts-exts bin/%name%.ts",
+  "watch": {
+    "include": "./**/*.ts",
```

Also, this is inconsistent with the other templates:

```suggestion
    "include": "**/*.ts",
```

Or change the other templates to use `"./**/sth"` instead of `"**/sth"`, but it should be consistent among them.